lib/build.api.js

```diff
@@ -110,13 +110,30 @@ class Build {
         this.aquifer.console.log(this.options.symlink ? 'Creating symlinks...' : 'Copying files and directories...');
         let links = [];
         let project = this.aquifer.project;
+        let destPaths = {
+          '7': {
+            'profiles': 'profiles/',
+            'drush': 'sites/all/drush',
+            'modules': 'sites/all/modules',
+            'themes': 'sites/all/themes',
+            'site': 'sites/default'
+          },
+          '8': {
+            'profiles': 'profiles',
+            'drush': 'drush',
+            'modules': 'modules',
+            'themes': 'themes',
+            'site': 'sites/default'
+          }
+        };
+        let versionDestPaths = destPaths[project.drupalVersion];
```

Comment:
I think this should be a class property, `this.paths.destinations` or something, simply because extensions will need to be able to interact with files without having to specify the Drupal version.